src/memray/_memray/record_writer.pxd

```diff
@@ -2,8 +2,12 @@ from _memray.sink cimport Sink
 from libcpp cimport bool
 from libcpp.memory cimport unique_ptr
 from libcpp.string cimport string
+from libcpp.vector cimport vector
 
 
 cdef extern from "record_writer.h" namespace "memray::api":
     cdef cppclass RecordWriter:
         RecordWriter(unique_ptr[Sink], string command_line, bool native_trace) except+
+
+cdef extern from "native_resolver.h" namespace "memray::native_resolver":
+    vector[string] unwindHere() except+
```

Comment:
Shouldn't we put this into a `native_resolver.pxd` instead? Seems weird to put it into `record_writer.pxd` when it lives in `native_resolver.h`